attestation-evidence-broker/src/main.rs

```diff
@@ -0,0 +1,69 @@
+use clap::{crate_authors, crate_version, App, Arg};
+use std::thread;
+use ttrpc_server::rpc_server;
+
+mod aeb_modules;
+mod device;
+mod protocols;
+mod ttrpc_server;
+
+#[macro_use]
+extern crate log;
+
+fn main() {
+    env_logger::builder()
+        .filter(None, log::LevelFilter::Info)
+        .init();
+
+    let matches = App::new("SEV Attestation Evidence Broker")
+        .version(crate_version!())
+        .author(crate_authors!())
+        .about("SEV Attestation Evidence Broker for provide evidence for guest attestation agent")
+        .arg(
+            Arg::with_name("listen")
+                .long("listen")
+                .short("l")
+                .help("Specify the socket listen addr. For example: vsock:///tmp/aeb.sock, unix:///tmp/aeb.sock")
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name("port")
+                .long("port")
+                .short("p")
+                .help("Specify the socket listen port. Default is 5577")
+                .takes_value(true),
+        )
+        .get_matches();
+
+    let sockaddr = match matches.is_present("listen") {
+        true => matches
+            .value_of("listen")
+            .expect("port is required")
+            .to_string(),
+        false => "vsock:///tmp/aeb.sock".to_string(),
+    };
+    let listen_port = match matches.is_present("port") {
+        true => matches
+            .value_of("port")
+            .expect("port is required")
+            .parse::<u16>()
+            .expect("port must be a valid integer"),
+        false => 5577,
+    };
+
+    let mut server = rpc_server::new_server(sockaddr, listen_port);
+    let _ = server.start().unwrap();
```

Comment:
We can change the form of the main function to `fn main() -> Result<()>` to eliminate the use of `unwrap` and `expect`.